packages/kmod-5.15-nvidia/kmod-5.15-nvidia.spec

```diff
@@ -1,10 +1,20 @@
-%global tesla_515 515.86.01
-%global tesla_515_libdir %{_cross_libdir}/nvidia/tesla/%{tesla_515}
-%global tesla_515_bindir %{_cross_libexecdir}/nvidia/tesla/bin/%{tesla_515}
-%global tesla_515_firmwaredir %{_cross_libdir}/firmware/nvidia/%{tesla_515}
+%global tesla_major 535
+%global tesla_minor 129
+%global tesla_patch 03
+%global tesla_ver %{tesla_major}.%{tesla_minor}.%{tesla_patch}
+%global tesla_535_libdir %{_cross_libdir}/nvidia/tesla/%{tesla_ver}
+%global tesla_535_bindir %{_cross_libexecdir}/nvidia/tesla/bin/%{tesla_ver}
+%global tesla_535_firmwaredir %{_cross_libdir}/firmware/nvidia/%{tesla_ver}
```

Comment:
Why not remove the major version from this var like we did for the others?

Reply:
We can, the driver seemed to be namespaced for the version and I wasn't sure if users counted on that namespacing in the existing driver build. For `kmod-5.10-nvidia` and `kmod-5.15-nvidia` the drivers are located in the versioned directory:
```shell
/usr/lib/nvidia/tesla/470.223.02/libEGL.so.1
...
/usr/lib/nvidia/tesla/535.129.03/libEGL.so.1
...
```
vs `kmod-6.1-nvidia` its just in `tesla`
```
/usr/lib/nvidia/tesla/libEGL.so.1
...
```
If we don't have anything depending on this version namespacing, then its far simpler to use the 6.1 paths.

Reply:
I was able to get the driver to work without the namespacing and now 5.15 and 6.1 work the same way.